source/_integrations/cloudflare.markdown

```diff
@@ -13,6 +13,16 @@ With the `cloudflare` integration, you can keep your Cloudflare records up to da
 
 The integration runs every hour, but can also be started manually by using the service `cloudflare.update_records` under services.
 
+<div class='note warning'>
+Due to a limitation in the Cloudflare API you can not use this integration with any of the folowing TLD's:
+
+- `.cf`
+- `.ga`
+- `.gq`
+- `.ml`
+- `.tk`
+</div>
+
```

Comment:
```suggestion
<div class='note warning'>

Due to a limitation in the Cloudflare API, you can not use this integration with any of the following TLD's:

- `.cf`
- `.ga`
- `.gq`
- `.ml`
- `.tk`

</div>

```

Typos and you need to make sure a new line between markdown content and the HTML content is present, or else the Markdown parse won't be able to switch language context.